src/build/args.rs

```diff
@@ -19,6 +19,12 @@ impl BuildArgs {
         matches!(self.subcommand, Some(BuildSubcommands::Web))
     }
 
+    /// Whether to build with optimizations.
+    #[cfg(feature = "wasm-opt")]
```

Comment:
Do we already want to hide this behind the `wasm-opt` feature flag here? Does it not make sense to build a release without wasm-opt?

Reply:
I think #199 may affect whether this is feature-flagged, but I'm not sure...

Reply:
I think this function would be marked as unused when `wasm-opt` is disabled, but I haven't tested it!

Reply:
That's right, the `--release` arg still exists without the feature flag, but we only use this accessor for `wasm-opt` right now.
Clippy complains if we don't feature gate it